crates/node/service/src/actors/network/actor.rs

```diff
@@ -62,7 +62,7 @@ pub struct NetworkInboundData {
     /// Handler for RPC Requests sent to the network actor.
     pub rpc: mpsc::Sender<P2pRpcRequest>,
     /// A channel to send unsafe blocks to the network actor.
```

Comment:
Maybe adjust this comment to note that this channel is for use by the sequencer / `admin` RPC only